modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterManagementGroupManager.java

```diff
@@ -179,6 +182,31 @@ public ClusterManagementGroupManager(
         scheduledExecutor = Executors.newSingleThreadScheduledExecutor(
                 NamedThreadFactory.create(clusterService.nodeName(), "cmg-manager", LOG)
         );
+
+        var messageCallback = new CmgMessageCallback() {
+            @Override
+            public void onClusterStateMessageReceived(ClusterStateMessage message, ClusterNode sender, @Nullable Long correlationId) {
+                assert correlationId != null;
```

Comment:
```suggestion
                assert correlationId != null : sender;
```

Reply:
How will this help?

Reply:
It will allow us to find out which node sent us `null`.

Reply:
I don't understand how this will help in any way, we have a lot of similar checks in other places. How are you going to use the sender node?

Reply:
I find out that some node sent a message not through Invoke.

Reply:
> I find out that some node sent a message not through Invoke.

And what will you do next? Anyway, I don't really care, let's add this, if you want